src/addrman.cpp

```diff
@@ -67,12 +67,12 @@ double CAddrInfo::GetChance(int64_t nNow) const
 
 CAddrInfo* CAddrMan::Find(const CNetAddr& addr, int* pnId)
 {
-    std::map<CNetAddr, int>::iterator it = mapAddr.find(addr);
+    auto it = mapAddr.find(addr);
```

Comment:
I'd rater see this changed to `std::map<CNetAddr, int>::const_iterator it = ...` for instance.

Reply:
The following `scripted-diff` compiles if someone wants to look for `::const_iterator` candidates:

```
git grep -lE '::iterator ' "*.cpp" "*.h" | \
    grep -vE '(crypto_tests.cpp|coins_tests.cpp|limitedmap.h|guiutil.cpp|addrman.cpp|bloom.cpp|net.cpp|net_processing.cpp|coins.cpp|coins.h)' | \
    xargs sed -i 's/::iterator /::const_iterator /g'
```